autopilot/prometheus.go

```diff
@@ -0,0 +1,66 @@
+package autopilot
+
+import (
+	"go.sia.tech/renterd/api"
+	"go.sia.tech/renterd/internal/prometheus"
+)
+
+type AutopilotStateResp api.AutopilotStateResponse
+
+func (asr AutopilotStateResp) PrometheusMetric() (metrics []prometheus.Metric) {
```

Comment:
I'd implement `PrometheusMetric` or maybe `ToPrometheus` on the actual return type in the `api` package within a single `api/prometheus.go`. That way we don't need 3 of these `prometheus.go` files with duplicated types.

If a return type doesn't exist yet because it's directly returning a `types.Currency` or something it's fine to create a new one in the `api` package.

Reply:
@n8maninger wrote this I can'ti would run it by him before I make any changes to it.